integration/spark/app/integrations/container/deltaMergeCommandVerificationStart.json

```diff
@@ -0,0 +1,154 @@
+{
+  "eventType": "START",
+  "inputs": [
```

Comment:
Shouldn't in this case `updates` table be an input as well?
I think we're doing it this way for `io.openlineage.spark3.agent.lifecycle.plan.MergeIntoCommandInputDatasetBuilder`

Reply:
updates table is in inputs :) also the events which is the target in merge command :)